package-lock.json

```diff
@@ -0,0 +1,4224 @@
+{
```

Comment:
You are pushing up your package-lock but not your package.json which means there are no scripts that I have access to run or test